cypress/integration/tests/applicationinventory/application/sort.test.ts

```diff
@@ -0,0 +1,125 @@
+/// <reference types="cypress" />
+
+import {
+    login,
+    clickByText,
+    sortAsc,
+    sortDesc,
+    verifySortAsc,
+    verifySortDesc,
+    getTableColumnData,
+} from "../../../../utils/utils";
+import { navMenu } from "../../../views/menu.view";
+import { applicationinventory, name, tagCount, review } from "../../../types/constants";
+
+import * as data from "../../../../utils/data_utils";
+import { ApplicationInventory } from "../../../models/applicationinventory/applicationinventory";
+
+var applicationsList: Array<ApplicationInventory> = [];
+
+describe("Business services sort validations", function () {
+    before("Login and Create Test Data", function () {
+        // Perform login
+        login();
+        var tagsList = ["C++", "COBOL", "Java"];
+        // Create multiple applications with tags
+        for (let i = 0; i < 3; i++) {
+            // Create new application
+            const application = new ApplicationInventory(data.getFullName(), "", "", "", tagsList);
+            application.create();
+            applicationsList.push(application);
+            tagsList.pop();
+        }
+    });
+
+    beforeEach("Persist session", function () {
+        // Save the session and token cookie for maintaining one login session
+        Cypress.Cookies.preserveOnce("AUTH_SESSION_ID", "KEYCLOAK_SESSION");
+
+        // Interceptors
+        cy.intercept("GET", "/api/application-inventory/application*").as("getApplications");
+    });
+
+    after("Perform test data clean up", function () {
+        // Delete the applications created before the tests
+
+        applicationsList.forEach(function (application) {
+            application.delete();
+        });
+    });
+
+    it("Name sort validations", function () {
+        // Navigate to application inventory page
+        clickByText(navMenu, applicationinventory);
+        cy.wait("@getApplications");
+
+        // get unsorted list when page loads
+        const unsortedList = getTableColumnData(name);
```

Comment:
I ran this test, but we need some other approach to pick column data, as I can see that, the identifier - td[data-label=Name] is not unique, there are other entries nested with same identifier. Hence, it is picking wrong values.